packages/mesos-modules/buildinfo.json

```diff
@@ -3,7 +3,7 @@
     "single_source" : {
       "kind": "git",
       "git": "https://github.com/dcos/dcos-mesos-modules.git",
-      "ref": "7ce5ba397bbaa8fdcc985ef2e6dd8bc81709931a",
-      "ref_origin": "1.9.x"
+      "ref": "2a5a02f1de47dcbe44844f7358adf85ea630cf4a",
```

Comment:
yeah would be good to have the commit in as well. Its only for tests, so won't affect the actual operation of the modules, but for completeness would be good to have that commit as well.